fixtures/react_native_windows_e2e_app/windows/RNWTestApp.sln

```diff
@@ -0,0 +1,140 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Do we need a solution file checked into source control? I don't see any absolute paths, so that may be fine for convenience

Reply:
Yes we do.